src/Microsoft.Data/Microsoft.Data.DataFrame.csproj

```diff
@@ -4,8 +4,6 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <LangVersion>7.3</LangVersion>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <!-- TODO: Remove this https://github.com/dotnet/corefxlab/issues/2724 -->
```

Comment:
Now that the `==` operators are being removed, we can delete this

Reply:
Let's close that issue when this gets checked in then. Now that warnings as errors are turned on, we shouldn't add any more warnings to this library.